### PR TITLE
Po/ init frontend and combine backend

### PR DESCRIPTION
**Pull request by BoddyShen**

### CODE REVIEW

**Comment by xyw0025** [comment]
image

seems like that last line in backend/.gitignore isnt doing anything here


[How to go back to previous directory in .gitignore：](https://stackoverflow.com/questions/67593309/how-to-go-back-to-previous-directory-in-gitignore)

image

@BoddyShen  let's see if we want another .gitignore file in the root directory

**Comment by BoddyShen** [comment]
Thanks,I got it! Let's add .gitignore file in the root directory.

**Comment by BoddyShen** [comment]
> image
> seems like that last line in backend/.gitignore isnt doing anything here
> 
> [How to go back to previous directory in .gitignore：](https://stackoverflow.com/questions/67593309/how-to-go-back-to-previous-directory-in-gitignore)
> 
> image
> @BoddyShen let's see if we want another .gitignore file in the root directory

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+### .idea ###
+.idea
```
Comment (xyw0025) [inline .gitignore]: new line at the end of file needed

File: backend/.gitignore [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+### .env ###
+.env
+
+### mvnw ###
+
+mvnw
+mvnw.cmd
+
+### .idea ###
+../.idea
```
Comment (xyw0025) [inline backend/.gitignore]:
```suggestion
```

line 57-58 are redundant. might as well remove them

**Review by xyw0025 — APPROVED** [review]
🅰️  looks good to me

**Pull request merged**